canvas_modules/harness/package.json

```diff
@@ -5,8 +5,10 @@
   "license": "Apache-2.0",
   "scripts": {
     "start": "NODE_ENV=development node index.js",
+    "cypress:run": "cypress open",
     "build": "npx stylelint 'src/**/*.scss' 'assets/**/*.scss' && grunt",
-    "test": "npx cypress open",
+    "test": "npx cypress open && npx cypress run",
```

Comment:
Why was this changed?

Reply:
I initially added the script to see if Cypress would open and run in the terminal simultaneously, but unfortunately, it didn't work as expected. I intended to revert the changes but it seems I have forgot to.

Reply:
I was not able to comment on your question above, but I did some more research and the .nycrc file configures NYC (Istanbul's CLI) for code coverage, specifying which files to include, exclude, and the reporting format. Webpack is used with Babel to instrument the code via babel-plugin-istanbul before bundling it. Here was the documentation for NYC [https://github.com/istanbuljs/nyc](url), it can also be found on the cypress docs.